internal/handlers/pg/pgdb/query.go

```diff
@@ -299,11 +300,11 @@ func prepareWhereClause(p *Placeholder, sqlFilters *types.Document) (string, []a
 						// type not supported for pushdown
 
 					case float64, bool, int32, int64:
-						filters = append(filters, fmt.Sprintf(sql, p.Next(), p.Next(), sjson.GetTypeOfValue(v)))
+						filters = append(filters, fmt.Sprintf(sql, p.Next(), p.Next(), commonparams.AliasFromType(v)))
```

Comment:
This is the part I don't understand. I see why it's safe to use `commonparams.AliasFromType` to show type when we return errors (we just return it, we don't need to compare etc), but in this place, I'm not sure if I understand the use case. Is it safe?